code/core/src/main/java/com/adobe/marketing/mobile/AdobeError.java

```diff
@@ -27,6 +27,18 @@ public class AdobeError implements Serializable {
     /** when a callback is null. */
     public static final AdobeError CALLBACK_NULL = new AdobeError("general.callback.null", 2);
 
+    /** when a server error happens. */
+    public static final AdobeError SERVER_ERROR =
+            new AdobeError("general.callback.server.error", 4);
+
+    /** when a network error happens. */
+    public static final AdobeError NETWORK_ERROR =
+            new AdobeError("general.callback.network.error", 5);
+
+    /** when an invalid request is made. */
+    public static final AdobeError INVALID_REQUEST =
+            new AdobeError("general.callback.invalid.request", 6);
```

Comment:
"general.request.invalid"

Reply:
Can you also add 
`public static final AdobeError INVALID_RESPONSE =
            new AdobeError("general.response.invalid", 7);`